### ISSUE TITLE
Babel Unknow option

### ISSUE THREAD

**Issue by kalehrishi** [jschr/electron-react-redux-boilerplate#7]
I am trying to setup boilerplate on `Windows 7` `Node 4.2.2`

``` bash

 npm ls --depth=0
electron-react-redux-boilerplate@0.0.0 E:\Workspace-Hrishikesh\electron-react-redux-boilerplate
├── babel-cli@6.3.13
├── babel-core@6.3.13
├── babel-eslint@4.1.6
├── babel-preset-es2015@6.3.13
├── babel-preset-stage-0@6.3.13
├── browser-sync@2.10.0
├── chai@3.4.1
├── electron-compile@1.0.1
├── electron-compilers@1.0.1
├── electron-packager@5.1.1
├── electron-prebuilt@0.34.5
├── eslint@1.10.3
├── eslint-config-airbnb@0.1.0
├── eslint-plugin-react@3.11.3
├── mocha@2.3.4
├── react@0.14.3
├── react-dom@0.14.3
├── react-redux@4.0.0
├── redux@3.0.4
├── redux-actions@0.8.0
└── redux-devtools@2.1.5

```

![image](https://cloud.githubusercontent.com/assets/517439/11612007/76ee9f7e-9c0d-11e5-9d4f-083392a4bc88.png)

I think this is babel error.

**Comment by kuka-radovan** [comment]
This happens because electron-compilers uses older babel implementation. They haven't switched to Babel 6 yet.

**Comment by nstehr** [comment]
I'm seeing the same issue on OSX El Capitan.  I reverted back to commit: 5a4e58467d5dbd0ba90ce5277d95bea66465d366 and it worked.  This issue appears to be related to the update to Babel 6.

**Comment by jschr** [comment]
I believe the issue is happening because electron-compile and electron-compilers is still using babel 5. I also think it might have something to do with npm3 because I wasn't seeing the error previously but now I do on a fresh install.

version 2 of electron-compile and electron-compilers should fix this and appears to be in active development so hopefully it will be released soon: 

https://github.com/electronjs/electron-compile/tree/electron-compile-2
https://github.com/electronjs/electron-compilers/tree/electron-compile-2

I tried the beta version but the api seems to have changed and the documentation isn't updated yet. As soon as it's released I'll make a fix.

**Comment by edsu** [comment]
I electron-compile v2.0 is out now?

**Comment by jschr** [comment]
thanks, will get a fix out asap

**Comment by senthilporunan** [comment]
Thanks for this boilerplate. is there any workaround ?

**Comment by jschr** [comment]
Sorry for the delay on this, tried working on this again a couple days ago but part of upgrading electron-compile also meant upgrading electron-prebuilt. There were a bunch of api changes and ran into some other issues trying to get then dependencies updated. I will try again when I get some downtime but unfortunately I'm quite busy right now. If there's anyone willing to take this on (or even the repo) let me know.

**Comment by jicksta** [comment]
@jschr Any progress on a fix for this? Still an issue on Mac OS X.

**Comment by jschr** [comment]
@jicksta unfortunately I haven't made any progress sorry. This boilerplate is probably due for a rewrite since a lot of the electron dependencies need to be updated to their new apis. I would gladly accept any PRs if anyone is up for it.

**Comment by orielz** [comment]
+1 (Windows OS)
Any workaround?

**Comment by javisperez** [comment]
+1  same bug here :-\ this boilerplate seems like what i need but can get it running

**Comment by dkarter** [comment]
I hate to +1 but any ideas how to solve this?

**Comment by squiles** [comment]
+1

**Comment by pronebird** [comment]
That should be fixed with #9

**Comment by jschr** [comment]
Closing since https://github.com/jschr/electron-react-redux-boilerplate/pull/9 was merged. Fixing packager is still a todo. A separate PR will be made for it.
